README.md

```diff
@@ -1,6 +1,89 @@
-# Nuxt Module
+<!--
+Get your module up and running quickly.
+
+Find and replace all on all files (CMD+SHIFT+F):
+- Name: My Module
+- Package name: my-module
+- Description: My new Nuxt module
+-->
+
+# My Module
+
+[![npm version][npm-version-src]][npm-version-href]
+[![npm downloads][npm-downloads-src]][npm-downloads-href]
+[![License][license-src]][license-href]
+
+> My new Nuxt module
+
+- [✨ &nbsp;Release Notes](/CHANGELOG.md)
+<!-- - [📖 &nbsp;Documentation](https://example.com) -->
+
+## Features
+
+<!-- Highlight some of the features your module provide here -->
+- ⛰ &nbsp;Foo
+- 🚠 &nbsp;Bar
+- 🌲 &nbsp;Baz
+
+## Quick Setup
+
+1. Add `my-module` dependency to your project
+
+```bash
+# Using yarn
+yarn add --dev my-module
+
+# Using npm
+npm install --save-dev my-module
```

Comment:
Might be nice to add `pnpm` as well

Reply:
I was torn to use `nr` because we don't know what package manager users are using, we could make the starter pnpm-first though 🤔

Reply:
Soon we will have smart `nuxt module add` (using nypm/corepack). Also docs can smartly convert this section. For now, i think adding classic 3 command would be good enough to move forward.

Reply:
Done in [0262799](https://github.com/nuxt/starter/pull/216/commits/02627994f644ee8d4ca9b36b5684752812d3f0a6)